src/BlockService/CheckoutBlockService.php

```diff
@@ -56,10 +52,10 @@ function () {
      */
     public function availableGateways()
     {
-        $currency = sanitize_text_field(wp_unslash($_POST['currency']));
-        $cartTotal = filter_input(INPUT_POST,'cartTotal', FILTER_SANITIZE_NUMBER_INT);
-        $paymentLocale = sanitize_text_field(wp_unslash($_POST['paymentLocale']));
-        $billingCountry = sanitize_text_field(wp_unslash($_POST['billingCountry']));
+        $currency = filter_input(INPUT_POST, 'currency', FILTER_SANITIZE_STRING);
+        $cartTotal = filter_input(INPUT_POST, 'cartTotal', FILTER_SANITIZE_NUMBER_INT);
+        $paymentLocale = filter_input(INPUT_POST, 'paymentLocale', FILTER_SANITIZE_STRING);
+        $billingCountry = filter_input(INPUT_POST, 'billingCountry', FILTER_SANITIZE_STRING);
```

Comment:
:thinking: 
why revert the PHP 8 deprecation fix?

Reply:
I must have messed it when merging different times.Now I've checked for all occurrences. Thanks!